ESPWebThingAdapter.h

```diff
@@ -233,6 +233,7 @@ class WebThingAdapter {
     while (device != nullptr) {
       JsonObject& descr = things.createNestedObject();
       this->serializeDevice(descr, device);
+      descr["href"] = "/things/" + device->id;
```

Comment:
The setting of descr["href"] should be in serializeDevice for consistency

Reply:
I moved it here so that it only gets included in the Things resource, not the Thing resource.

Reply:
:+1: